core/src/edu/asu/c3simulator/screens/BusinessCreationGuideScreens.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.c3simulator.screens;
+
+import java.util.Hashtable;
+import java.util.Set;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+
+import edu.asu.c3simulator.widgets.Location;
+
+/**
+ * Parents class containing methods that interact with the business creation guide.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class BusinessCreationGuideScreens
```

Comment:
nitpick: Should be "...Screen" (singular), unless it is a container for all of the screens